audio/build.zig

```diff
@@ -0,0 +1,119 @@
+const std = @import("std");
+const Builder = std.build.Builder;
+
+const soundio_path = thisDir() ++ "/upstream/soundio";
+
+pub const pkg = std.build.Pkg{
+    .name = "soundio",
+    .source = .{ .path = thisDir() ++ "/soundio/main.zig" },
+};
```

Comment:
this should point to `audio/main.zig` right? We can do this in another PR, just making a note